apps/gallery/index.html

```diff
@@ -105,7 +105,7 @@ <h1 id="pick-header-title" data-l10n-id="pickoneimage2"></h1>
     </section>
 
     <section role="region" id="fullscreen-view" class="skin-dark">
-      <gaia-header id="fullscreen-toolbar-header" action="back" class="large-only">
+      <gaia-header id="fullscreen-toolbar-header" action="back" class="large-only" no-font-fit>
```

Comment:
Yes, font-fit will be done no matter it's displayed or not :(

Added |title-start="50"|.